README.md

```diff
@@ -20,12 +20,20 @@ Whether you just want to show a license or see its human-readable explanation, y
 
  - `#license-<license>` (hash): set the license to display
  - `fullname=<name>`: set the copyright holder name
- - `year=<name>`: set the starting copyright yea
+ - `year=<name>`: set the starting copyright year
  - `hide_explanations=<true|false>` (default: `false`): hide the explanations column
 
 
 [![showalicense.com](http://i.imgur.com/AB42LJo.png)](http://showalicense.com/)
 
+## Developer installation
+
+- `$ npm i -g browserify watchify blah`;
```

Comment:
Watchify is not needed (it's already installed in the dev dependencies). It's only me (the npm publisher) who uses blah, also.

Not sure if `browserify` is needed... Just `npm run watch` and `npm start` should work.

Reply:
I do not understand how large is blah's role. At the beginning of `README.md` is this comment:

```html
<!-- Please do not edit this file. Edit the `blah` field in the `package.json` instead. If in doubt, openan issue. -->
```

I tried running

```
$ blah -r
```

but it does not put in the generated `README.md` file, [this][1] part of the blah section from `package.json`. Exactly this part of my contribution motivated me to include blah in the developer installation instructions because, in future, edits to `README.md` might be needed and manual editing of `README.md` is apparently forbidden. Please tell me if I made myself understood or not.

  [1]: https://github.com/silviubogan/showalicense.com/blob/679e0282abae4a2cbe93cfe83327705bc53e4988/package.json#L42-L53

Reply:
Yeah, because it's using my config files. It's only me generating the docs when publishing to `npm`. 

https://ionicabizau.net/blog/33-how-i-npm

So, code contributors don't really need to touch the `md` files because they are generated anyways on publish.

Reply:
Thanks for the link. Is the code OK in 757be94?